src/modules/authentication/api/authentication.controller.ts

```diff
@@ -93,7 +93,9 @@ export class AuthenticationController {
     @ApiUnauthorizedResponse({ description: 'User is not logged in.' })
     @ApiOkResponse({ description: 'Returns info about the logged in user.', type: UserinfoResponse })
     public async info(@AuthenticatedUser() user: User): Promise<UserinfoResponse> {
-        const person: Person<true> | null | undefined = await this.personRepository.findByKeycloakUserId(user.sub);
-        return new UserinfoResponse(user, person);
+        const personPermissions: PersonPermissions = await this.personPermissionsRepo.loadPersonPermissions(user.sub);
+        const roleIds: string[] = await personPermissions.getRoleIds();
+        this.logger.info(roleIds.toString());
```

Comment:
Soll das wirklich in prod auf info level geloggt werden, oder war das nur fürs testing?

Reply:
Kommt mir auch ein wenig spammy vor

Reply:
Absolut, das habe ich vergessen, danke!